source/docs/appendices/migration-guides.md

```diff
@@ -26,6 +26,55 @@ The properties that were tagged as deprecated in `theme/components/atoms/Button/
 
 Please update your code to fit the new behavior.
 
+### Updated the Prismic Module
+
+#### Enforce trailing slash
```

Comment:
This doesn't belong to the migration guide I think. It should be documented under the Prismic section, and the migration guide could contain direct links to those pages.

See https://developers.front-commerce.com/docs/appendices/migration-guides.html#New-features-in-2-13-0 for examples.

Reply:
Yea it's quite confusing finding the right place and sync when it doesn't fall under the same cycle. 📦 soon 👯

Reply:
I cleaned up the migration guide, and improved the routable type docs